org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -964,6 +987,185 @@ public void recyclableThreads(ThreadReference thread) {
             return new Responses.VariablesResponseBody(list);
         }
 
+        Responses.ResponseBody setVariable(Requests.SetVariableArguments arguments) {
+            Map<String, Object> options = new HashMap<>();
+            // TODO: when vscode protocol support customize settings of value format, showQualified should be one of the options.
+            boolean showStaticVariables = true;
```

Comment:
This should be false by default. User will need to explicitly turn it on by configuring launch.json. If your purpose is to increase test coverage, then I agree that we can keep this value until we the official release.